qmetaobject_impl/src/lib.rs

```diff
@@ -1,4 +1,5 @@
-/* Copyright (C) 2018 Olivier Goffart <ogoffart@woboq.com>
+/* Original work Copyright (C) 2018 Olivier Goffart <ogoffart@woboq.com>
+   Modified work Copyright (C) 2020 ivan tkachenko a.k.a. ratijas <me@ratijas.tk>
```

Comment:
Put two "Copyright"   lines,  no need to include Original or Modified

Reply:
About that...
I'm not sure how to do it properly, because I lack any experience in that field.
Copyright sure serves as a pointer, and in a case of emergency it says "that guy may be responsible for it, go ask him/her", but we already have bit blame for that. Also I don't want to get in a way in case of relicensing etc.
Microsoft and some others handle it by requiring contributors to sign [CLA](https://cla.opensource.microsoft.com/MicrosoftDocs/win32) which grants "irrevocable license in the Submission" etc etc etc.